src/commands/derive_parser.rs

```diff
@@ -0,0 +1,27 @@
+use crate::utils::auth_token::AuthToken;
+use crate::utils::value_parsers::kv_parser;
+use clap::{command, value_parser, ArgAction::SetTrue, Parser, Subcommand};
+
+#[derive(Parser)]
+pub(super) struct SentryCLI {
+    #[command(subcommand)]
+    pub(super) command: SentryCLICommand,
+
+    #[arg(global=true, long="header", value_name="KEY:VALUE", value_parser=kv_parser, help="Custom headers that should be attached to all requests{n}in key:value format")]
+    pub(super) headers: Vec<(String, String)>,
+
+    #[arg(global=true, long, value_parser=value_parser!(AuthToken), help="Use the given Sentry auth token")]
+    pub(super) auth_token: Option<AuthToken>,
+
+    #[arg(global=true, ignore_case=true, long, value_parser=["trace", "debug", "info", "warn", "error"], help="Set the log output verbosity")]
+    pub(super) log_level: Option<String>,
```

Comment:
Same here, would be cool if we can use `Option<&'static str>`. Will need some code that calls this to check if possible